academy/static/src/js/animal.js

```diff
@@ -0,0 +1,26 @@
+odoo.define('academy.animal', function (require){
+    'use strict';
+
+    var classTest = require('web.Class');
```

Comment:
use a better name for the require

```suggestion
    var WebClass = require('web.Class');
```